resources/ext/ezbake.conf

```diff
@@ -8,6 +8,19 @@ ezbake: {
    pe: {}
    foss: {
       redhat: { dependencies: ["puppet-agent >= 1.6.0"],
+                build-dependencies: ["%{open_jdk}"],
+                # Install some gems
+                install: [
+                  "pwd",
+                  "ls -l",
+                  "ls -l ext",
+                  "ls -l ext/build-scripts",
+                  "cat ./ext/build-scripts/foo.sh",
+                  "bash ./ext/build-scripts/foo.sh",
```

Comment:
This is just a bunch of debugging stuff, except for the last line, which calls our build script.  I've run this through ezbake and verified that the build script gets called correctly and produces the appropriate output on both EL and debian platforms.